### PR TITLE
iwa: Add --internal-routing flag for app creation

### PR DESCRIPTION
**Pull request by watsonian**
This adds the requisite flag for an upcoming feature. The API changes for this have yet to be implemented and this may change a bit depending on https://github.com/heroku/dogwood/issues/1743. Just getting this up for general review and so we have something to reference for the changeset.

### CODE REVIEW

File: src/commands/apps/index.js [hunk 1]
```diff
@@ -93,6 +109,7 @@ theirapp   other@owner.name`,
     {name: 'json', description: 'output in json format'},
     {name: 'space', char: 's', hasValue: true, description: 'filter by space', completion: SpaceCompletion},
     {name: 'personal', char: 'p', description: 'list apps in personal account when a default team is set'},
+    {name: 'internal', char: 'i', description: 'filter to Internal Web Apps'},
```
Comment (jdxcode) [inline src/commands/apps/index.js]: should this be a hidden flag for now?

Comment (watsonian) [reply src/commands/apps/index.js]: Ah, yeah, it should.

**Comment by jdxcode** [comment]
please don't merge this because this repo has special commit rules to get it into the CHANGELOG. Just let me know when you're ready and I'll merge it.

**Comment by watsonian** [comment]
@jdxcode Okay, this should be ready to merge. The API changes we need from https://github.com/heroku/api/pull/9516 are merged and deployed. I missed something that's getting fixed in https://github.com/heroku/api/pull/9546, but the field is there so references to it will work properly.

**Pull request merged**